### PR TITLE
Fix docs barticks

### PR DESCRIPTION
**Pull request by neutrinoceros**
## PR Summary

follow up to #3264 with a pre-commit hook for future-proofing

### CODE REVIEW

File: doc/source/cookbook/complex_plots.rst [hunk 1]
```diff
@@ -39,42 +39,42 @@ is inherited by
...
+3. ``dpi``, i.e. the dots-per-inch in your final file, which can also
 be thought of as the actual resolution of your image.  This can
-only be set on save via the `mpl_kwargs` parameter to
+only be set on save via the ``mpl_kwargs`` parameter to
 :meth:`~yt.visualization.plot_container.PlotContainer.save`.  The
-`dpi` and `figure_size` together set the true resolution of your
-image (final image will be `dpi` :math:`*` `figure_size` pixels on a
-side), so if these are set too low, then your `buff_size` will not
+``dpi`` and ``figure_size`` together set the true resolution of your
+image (final image will be ``dpi`ù :math:`*` ``figure_size`` pixels on a
```
Comment (chummels) [inline doc/source/cookbook/complex_plots.rst:62]: Not sure why the prehook stuck a ``ù`` here, but I think it just needs to swap out to be a `.

```suggestion
image (final image will be ``dpi`` :math:`*` ``figure_size`` pixels on a
```

**Review by chummels — APPROVED** [review]
This looks great.  I checked it over and all the corrections seem valid except the one I noted.  Thanks for running this!

File: doc/source/cookbook/complex_plots.rst (hunk 1 above)
Comment (neutrinoceros) [reply doc/source/cookbook/complex_plots.rst:62]: The hook merely reported stuff, then I used my clumsy human hands to fix it. The err is human :)
Thanks for catching it !

Comment (neutrinoceros) [reply doc/source/cookbook/complex_plots.rst]: Fun fact: French keyboards have a `ù` alt key, though there's exactly one word in French that uses it: "où" (where).

**Pull request merged**